### PR TITLE
Sound visualizer

### PR DESCRIPTION
**Pull request by LucyWilcox**

### CODE REVIEW

**Comment by YuzhongHuang** [comment]
Looks good to me

File: app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java [hunk 1]
```diff
@@ -43,6 +43,8 @@
     private ArrayList voiceInput;
     private Intent recognizerIntent;
     private SpeechRecognizer sr;
+    private ViewGroup.LayoutParams cParams = new ViewGroup.LayoutParams(50, 50);
```
Comment (davidabrahams) [inline app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java]: remove this and test?

File: app/src/main/res/layout/fragment_speech.xml [hunk 2]
```diff
@@ -57,12 +56,19 @@
         android:paddingTop="5dp"
         android:paddingBottom="5dp"/>
 
-    <ImageView
-        android:layout_width="120dp"
-        android:layout_height="120dp"
-        android:id="@+id/listenButton"
+    <FrameLayout
+        android:layout_width="150dp"
```
Comment (davidabrahams) [inline app/src/main/res/layout/fragment_speech.xml]: Check this value and comment why it is what it is.

**Comment by LucyWilcox** [comment]
@greneroom @nmohamed Good to merge?

**Comment by nmohamed** [comment]
i haven't tested it but if it works then merge! code looks good

**Pull request merged**